.github/workflows/conan-package-release.yml

```diff
@@ -2,6 +2,92 @@ name: Create Conan Package Release
 
 on:
   workflow_dispatch:
+    organization:
+      description: 'Organization'
+      required: false
+      default: Ultimaker
+      type: string
+
+    repository:
+      description: 'repository'
+      required: true
+      default: 'Cura'
```

Comment:
Why have a `required` and a `default`? It seems a bit redundant to mark `repository` as required when it can default to Cura